base/cvd/BUILD.libvpx.bazel

```diff
@@ -17,12 +17,14 @@ genrule(
     ],
     cmd = " && ".join([
         "INITIAL_WD=`pwd`",
+        "AS_PATH=$$(dirname $$(realpath $(location @nasm)))",
         "cd `dirname $(location configure)`",
-        "./configure",
+        "env PATH=$$AS_PATH:$$PATH ./configure",
         "make",
         "cp libvpx.a $$INITIAL_WD/$(RULEDIR)",
     ]),
     visibility = ["//visibility:private"],
+    tools = ["@nasm"],
```

Comment:
This is the only place AFAIK that `nasm` is used. Could you remove the dependency from the control file (base/debian/control) and the github workflows (.github/workflows/*.yaml)?